docs/source/classes/Tutorials/1_Qubit/16a_JaynesCummingsHamiltonianBasis.ipynb

```diff
@@ -41,11 +41,11 @@
     "\n",
     "- $H_{JC} = \\hbar\\omega_{c} a^{\\dagger}a\\otimes 1_{2,2} + \\frac{1}{2}\\hbar\\omega_{q}1_{d,d}\\otimes\\sigma_{z} + \\hbar g(a^{\\dagger}\\otimes\\sigma_{-} + a\\otimes\\sigma_{+})$\n",
     "\n",
-    "    ![Cavity first and qubit with sigma-plus/minus operators and plus](CavityQubitWithSandPlus.png){ width=70% height=70% }\n",
+    "    ![Cavity first and qubit with sigma-plus/minus operators and plus](CavityQubitWithSandPlus.png)\n",
     "\n",
     "- $H_{JC} = \\hbar\\omega_{c}1_{2,2} \\otimes a^{\\dagger}a  + \\frac{1}{2}\\hbar\\omega_{q}\\sigma_{z}\\otimes 1_{d,d} + \\hbar g(\\sigma_{-}\\otimes a^{\\dagger} + \\sigma_{+}\\otimes a)$\n",
     "\n",
-    "    ![Qubit first and qubit with sigma-plus/minus operators and plus](QubitCavityWithSandPlus.png){ width=70% height=70% }\n"
```

Comment:
same as the previous and two more similar places below.

Reply:
Fixed